src/core/scene/flags.js

```diff
@@ -10,6 +10,7 @@
 
         picking: true,              // Picking enabled
         clipping: true,             // User-defined clipping enabled
+        frontClippingOnly: false,        // Used to assist drawing clipping caps
```

Comment:
@shrekshao can we rename this property to "clippingBackfaces" and invert its behaviour, so that:

* true by default
* ignored when **clipping** is false (current behaviour)
* when both **clipping** is true and **clippingBackfaces** false, then frontfaces are clipped and backfaces are not
* when **clipping** is true and **clippingBackfaces** is true, then both front and backfaces are clipped.